app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt

```diff
@@ -284,7 +299,7 @@ class Message : RealmObject {
         ) {
             false
         } else {
-            _isFullyDownloaded
+            areHeavyDataFetched
```

Comment:
Would we want to rename the method here to match the new `areHeavyDataFetched` naming as well?

Reply:
No, the `areHeavyDataFetched` is just a part of the `isFullyDownloaded` logic.